examples/hosted-delivery-function-potterdb-rest-api/README.md

```diff
@@ -0,0 +1,86 @@
+# Hosted Delivery Function - PotterDB
+This is an example of a hosted delivery function that uses the PotterDB REST API to enrich
+entries with data from the Harry Potter universe. Even though PotterDB has a GraphQL API
+itself, we use the REST API to showcase how to potentially wrap any API.
+
+This project was bootstrapped with [Create Contentful App](https://github.com/contentful/create-contentful-app).
+
+## How to use
+
+Execute create-contentful-app with npm, npx or yarn to bootstrap the example:
+
+```bash
+# npx
+npx create-contentful-app --typescript
```

Comment:
Do we want to make these example specific, e.g.
```suggestion
npx create-contentful-app --example hosted-delivery-function-potterdb-rest-api
```

Reply:
Then we (or @contentful/team-developer-experience) should probably look a bit more holistically at those files.